Filter/NodeSassFilter.php

```diff
@@ -142,4 +171,30 @@ public function filterLoad(AssetInterface $asset)
     public function filterDump(AssetInterface $asset)
     {
     }
+
+    protected function extractAndSaveSourceMap($output, $mapFileName)
```

Comment:
Why protected?

Reply:
I tend to use protected by default for internal methods since it opens up the possibility for developers to change the behaviour without cluttering the API of the class with public methods. I can change it to private if you rather use private.